apps/publications/src/app/components/year-range/year-range.component.ts

```diff
@@ -1,33 +1,35 @@
-import { Component, Input, OnInit } from '@angular/core';
-import { UntypedFormControl } from '@angular/forms';
+import { Component, Input, OnInit, ViewEncapsulation } from '@angular/core';
+import { FormControl } from '@angular/forms';
 import { MatDatepicker } from '@angular/material/datepicker';
+import { Moment } from 'moment';
 
 @Component({
   selector: 'perun-web-apps-year-range',
   templateUrl: './year-range.component.html',
   styleUrls: ['./year-range.component.scss'],
+  encapsulation: ViewEncapsulation.None,
 })
 export class YearRangeComponent implements OnInit {
-  @Input() startYear: UntypedFormControl;
-  @Input() endYear: UntypedFormControl;
+  @Input() startYear: FormControl<Moment>;
+  @Input() endYear: FormControl<Moment>;
   startMaxYear: Date;
   endMinYear: Date;
   endMaxYear: Date;
 
   ngOnInit(): void {
     this.endMaxYear = new Date();
-    this.startMaxYear = this.endYear.value as Date;
-    this.endMinYear = this.startYear.value as Date;
+    this.startMaxYear = this.endYear.value.toDate();
+    this.startYear.setValue(null);
```

Comment:
By setting this to null, it can cause this and parent component to call `.toDate()` on null